lib/hal/video.c

```diff
@@ -230,6 +236,24 @@ BOOLEAN XVideoListModes(VIDEO_MODE *vm, int bpp, int refresh, void **p)
 		if(pVidMode->refresh != refresh)
 			continue;
 
+		if(dwAdapter == AV_PACK_HDTV)
+		{
+			is_hd = pVidMode->dwMode & 0x80000000;
+
+			if(pVidMode->height == 720 && !allow_720p)
+				continue;
+
+			if(pVidMode->height == 1080 && !allow_1080i)
+				continue;
+
+//			If 480p support is enabled, reject 480i modes
+//			(forces a 480p mode to be picked) */
```

Comment:
- Comment indentation is weird.
- This has `// comment` but also `comment */` without `/*`
- Rest of file also seems to use `/* comment */`

Reply:
Fixed.
I used `/*  */` at first  but started looking at https://github.com/XboxDev/nxdk/blob/554da020e0805990b552d2c015b512aaa67640be/lib/hal/video.c#L378 and drew bad conclusions.